src/release/crater.md

```diff
@@ -0,0 +1,86 @@
+# Crater runs
```

Comment:
Shouldn't the title match the summary? Instead of having `Running crater` as a subtitle.